### DISCUSSION TITLE
Add Support to event mouse ZMK &mmv and &msc

### DISCUSSION THREAD

**Discussion by mh4x0f** [nickcoutsos/keymap-editor#206] [Ideas]
Hi, this is a nice ideia to add support to zmk mouse actions like move and scroll functions.

**Comment by mh4x0f** [comment]
duplicated

**Comment by nickcoutsos** [comment]
From what I've seen the movement behaviors aren't merged yet. You can enable it in the experimental features in the settings dialog but this assumes that you're building from a ZMK fork that includes it and won't adjust your build configuration for you. See [Keymap Editor Wiki: Experimental ZMK Features](https://github.com/nickcoutsos/keymap-editor/wiki/Experimental-ZMK-Features#mouse-emulation).
